docs/routing-rules.md

```diff
@@ -3,8 +3,16 @@
 Trino Gateway includes a routing rules engine.
 
 By default, Trino Gateway reads the `X-Trino-Routing-Group` request header to
-route requests. If this header is not specified, requests are sent to the
-default routing group called `adhoc`.
+route requests. If this header is not specified, requests are sent to the default routing group.
+
+You can configure a default routing group by setting the `defaultRoutingGroup` under the `routing` section. 
+This group will be used whenever routing information is unavailable or the external routing service fails. 
+If not configured, the fallback remains the built-in adhoc group.
```

Comment:
```suggestion
If not configured, the fallback remains the built-in group `adhoc`.
```
The name of the group is `adhoc`, o it is best practice to refer to it as the group `adhoc`  to avoid ambiguity.